### ISSUE TITLE
Add command to vscode command list

### ISSUE THREAD

**Issue by ebolefeysot** [jaydenlin/google-complete-me#1]
Hi. Very interesting extension. Seems to work ok so far.
Just one thing: you must insert the activation command (shift-ctrl-i) to the vscode command list.
The first thing I did after installing is 'F1 google' and nothing was available. I expect a command like 'Enable Google-complete'.
Also please add the shortcut to the settings (custom shortcut).
Thanks

**Comment by jaydenlin** [comment]
Hi Qwench,
Thank you for the feedback!
I just released 1.0.3 and added the command to the list. I named it as  "Google Complete Me: toggle auto-complete". If you have any feedback, please feel free to contact me. Thank you!
```
"commands": [
      {
        "command": "extension.googleCompleteMe",
        "title": "Google Complete Me: toggle auto-complete"
      }
    ]
```
https://github.com/jaydenlin/google-complete-me/commit/7d7dbcd6cf2965f5f0b1d85b6a383ec70572f498

**Comment by jaydenlin** [comment]
Hi Qwench,
Thank you for the feedback!
I just released 1.0.4 for this issue.

> Just one thing: you must insert the activation command (shift-ctrl-i) to the vscode command list.

I added the command to the list.

```
"commands": [
      {
        "command": "extension.googleCompleteMe",
        "title": "Google Complete Me: toggle auto-complete"
      }
    ]
```

> Also please add the shortcut to the settings (custom shortcut).

I am not sure how to add it.. so I only updated the README.md file [here](https://github.com/jaydenlin/google-complete-me#keybinding) for the shortcut setting.
Is it okay or is there another better way?

If you have any concern please feel free to leave comments here.
Thank you!
